### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by WagokiBry**
**Portfolio setup and mobile first**

For the first milestone in the process of creating my portfolio website, I will:

•set up a new repository and prepare it for development using best practices by implementing HTML&CSS linters
•build the first 2 sections of the mobile website using the template you chose from Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <h2 class="cap">LET’S CONNECT</h2>
+          </section>
+          
+<section class="img">
+  <ul>
+    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+    <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+    <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+    <li><a href="#"><i class='fab fa-github'></i></a></li> 
+    <li><a href="#"><i class='fab fa-angellist'></i></a></li>
```
Comment (ha-manel) [inline index.html]: - [ ] As requested by the previous reviewer, kindly make these icons look more like the ones in Figma, this includes the color, the spacing between them and the alignment of the list 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @PeacefulWiser 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
- [ ] From now on kindly avoid creating new PRs for re-reviews, even if you get "required changes", please make the changes in the same branch and submit the same pull request to another review 👍
- [ ] As requested by the previous reviewer, kindly use the correct `font-family` as shown in Figma. You can import the font from google fonts (please check out this [link](https://fonts.google.com/specimen/Poppins?query=poppins)) or you can use a cdn.

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header class="header">
+            <nav>
+           
+            <a href="#">My Logo</a>
```
Comment (jerryowusu) [inline index.html:20]: [OPTIONAL] Kindly put the logo outside the nav tag since it is not a navigation menu.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url("https://user-images.githubusercontent.com/66992901/198218680-01a4bf79-49e0-4ca5-b5a4-363c351466a5.png");
+  background-repeat: no-repeat;
+}
+
```
Comment (jerryowusu) [inline style.css:9]: There are some few disparities between your design and that of the figma. Kindly check and fix the white/blank space on the right side of your design. Also check the fonts and spacing to make sure it matches the design.

 Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-10-28 06-11-21  |  Screenshot from 2022-10-28 05-25-32

**Review by jerryowusu — APPROVED** [review]
Hi @PeacefulWiser 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

wow approved

## Highlights
All linters are passing :heavy_check_mark: 
Followed Github flow :heavy_check_mark: 
Good commit messages :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**